reconcile/utils/aws_api.py

```diff
@@ -637,11 +897,9 @@ def delete_keys(
                 ]
                 if not user_and_user_keys:
                     continue
-                # unpack single item from sequence
-                # since only a single user can have a given key
-                [user_and_user_keys] = user_and_user_keys
-                user = user_and_user_keys[0]
-                user_keys = user_and_user_keys[1]
+
+                user = user_and_user_keys[0][0]
+                user_keys = user_and_user_keys[0][1]
```

Comment:
there was an implicit check that `user_and_user_keys` has single item

Reply:
I've added an explicit check in [f446ead](https://github.com/app-sre/qontract-reconcile/pull/4841/commits/f446eadf03c2ec694c0f115f5d51db4b51d61812)